Compilation issue building Tpm2Tester on Linux

Not all targeted .net versions for Tpm2Tester support the overloads of [String.Contains](https://docs.microsoft.com/en-us/dotnet/api/system.string.contains?view=netframework-4.7.1#System_String_Contains_System_String_System_StringComparison_):

`TSS.MSR/Tpm2Tester/TestSubstrate/TestFramework.cs(508,94): error CS1501: No overload for method 'Contains' takes 2 arguments [~/src/TSS.MSR/Tpm2Tester/TestSubstrate/TestSubstrate.csproj]`

This breaks the Linux build of Tpm2Tester, because this call is hidden behind `#if __NETCOREAPP2__` (which is set for Linux, or if the target framework contains the string "2." or "3.")

```
$ dotnet --version
5.0.300

$ cd ~/src/TSS.MSR/Tpm2Tester && dotnet build
Microsoft (R) Build Engine version 16.10.0+4242f381a for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.2' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp3.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
  Restored ~/src/TSS.MSR/Tpm2Tester/TpmProxy/TpmProxy.csproj (in 197 ms).
  Restored ~/src/TSS.MSR/Tpm2Tester/TestSubstrate/TestSubstrate.csproj (in 197 ms).
  Restored ~/src/TSS.MSR/Tpm2Tester/TestSuite/TestSuite.csproj (in 1.36 sec).
  Restored ~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj (in 5.22 sec).
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.2' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp3.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/net472/TSS.Net.dll
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/net48/TSS.Net.dll
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/netstandard2.1/TSS.Net.dll
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/netcoreapp2.2/TSS.Net.dll
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/netcoreapp3.0/TSS.Net.dll
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/netcoreapp2.1/TSS.Net.dll
  TSS.Net -> ~/src/TSS.MSR/TSS.NET/bin/TSS.Net/Debug/netcoreapp3.1/TSS.Net.dll
~/src/TSS.MSR/Tpm2Tester/TestSubstrate/TestFramework.cs(508,94): error CS1501: No overload for method 'Contains' takes 2 arguments [~/src/TSS.MSR/Tpm2Tester/TestSubstrate/TestSubstrate.csproj]
  TestSubstrate -> ~/src/TSS.MSR/Tpm2Tester/bin/TestSubstrate/Debug/netcoreapp2.1/Tpm2TesterFramework.dll
~/dotnet-5.0/sdk/5.0.300/Microsoft.Common.CurrentVersion.targets(2202,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "BouncyCastle.Crypto, Version=1.8.3.0, Culture=neutral, PublicKeyToken=0e99375e54769942". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [~/src/TSS.MSR/Tpm2Tester/TpmProxy/TpmProxy.csproj]
  TpmProxy -> ~/src/TSS.MSR/Tpm2Tester/bin/TpmProxy/Debug/netcoreapp2.1/TpmProxy.dll

Build FAILED.

~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.2' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp3.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp2.2' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
~/dotnet-5.0/sdk/5.0.300/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp3.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [~/src/TSS.MSR/TSS.NET/TSS.Net/TSS.Net.csproj]
~/dotnet-5.0/sdk/5.0.300/Microsoft.Common.CurrentVersion.targets(2202,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "BouncyCastle.Crypto, Version=1.8.3.0, Culture=neutral, PublicKeyToken=0e99375e54769942". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [~/src/TSS.MSR/Tpm2Tester/TpmProxy/TpmProxy.csproj]
~/src/TSS.MSR/Tpm2Tester/TestSubstrate/TestFramework.cs(508,94): error CS1501: No overload for method 'Contains' takes 2 arguments [~/src/TSS.MSR/Tpm2Tester/TestSubstrate/TestSubstrate.csproj]
    5 Warning(s)
    1 Error(s)

Time Elapsed 00:00:10.99
```

--- a/Tpm2Tester/TestSubstrate/TestFramework.cs
+++ b/Tpm2Tester/TestSubstrate/TestFramework.cs
@@ -505,7 +505,7 @@ namespace Tpm2Tester
                     case TpmDeviceType.tbs:
                     case TpmDeviceType.tbsraw:
 #if __NETCOREAPP2__
-                        if (!System.Runtime.InteropServices.RuntimeInformation.OSDescription.Contains("Windows", StringComparison.InvariantCultureIgnoreCase))
+                        if (!System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Windows))
                             underlyingTpmDevice = new LinuxTpmDevice();
                         else
 #endif

